### ISSUE TITLE
turtle.write doc missing tuple parens for font default in 3.10+

### ISSUE THREAD

**Issue by 06ee7cf5-5a74-46d9-ad2e-ae3b574be929** [python/cpython#89560]
BPO | [45397](https://bugs.python.org/issue45397)
--- | :---
Nosy | @birkenfeld, @terryjreedy, @serhiy-storchaka

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2021-10-07.03:12:17.930>
labels = ['3.11', 'type-bug', '3.10', 'docs']
title = 'turtle.write doc missing tuple parens for font default in 3.10+'
updated_at = <Date 2021-11-27.02:54:27.094>
user = 'https://bugs.python.org/jggammon'
```

bugs.python.org fields:
```python
activity = <Date 2021-11-27.02:54:27.094>
actor = 'terry.reedy'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2021-10-07.03:12:17.930>
creator = 'jggammon'
dependencies = []
files = []
hgrepos = []
issue_num = 45397
keywords = []
message_count = 7.0
messages = ['403348', '403351', '403352', '403413', '403415', '403416', '407128']
nosy_count = 5.0
nosy_names = ['georg.brandl', 'terry.reedy', 'docs@python', 'serhiy.storchaka', 'jggammon']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue45397'
versions = ['Python 3.10', 'Python 3.11']
```

</p></details>

**Comment by slateny** [comment]
Seems fixed in [3.11](https://docs.python.org/3.11/library/turtle.html#turtle.write) but not in [3.10](https://docs.python.org/3.10/library/turtle.html#turtle.write)

**Comment by terryjreedy** [comment]
I verified that in 3.9 and 3.11, turtle.py, line 1098, is now rendered correctly.
`.. function:: write(arg, move=False, align="left", font=("Arial", 8, "normal"))`
https://docs.python.org/3.11/library/turtle.html#turtle.write
`turtle.write(arg, move=False, align='left', font=('Arial', 8, 'normal'))`

In 3.10, same line
`.. function:: write(arg, move=False, align="left", font=("Arial", 8, "normal"))`
renders without the tuple parens.
https://docs.python.org/3.10/library/turtle.html#turtle.write
`turtle.write(arg, move=False, align='left', font='Arial', 8, 'normal')`

If there is no obvious fix, I think that this should be closed.  But...

@JulienPalard Are we using a different version of Sphinx in 3.10 than in 3.9 and 3.11, or is some configuration setting different?

**Comment by JulienPalard** [comment]
Yes, see [docsbuild-scripts' README](https://github.com/python/docsbuild-scripts/#check-current-version=) (but it sometimes lag behind a few days), here's an up-to-date version:

- In 3.9 we use Sphinx==2.4.4
- In 3.10 we use Sphinx==3.2.1
- In 3.11 we use Sphinx==4.5.0

(It can also be checked by looking at the footers.)

**Comment by JulienPalard** [comment]
Closing this as I see it fixed in /3.8/ to /dev/.
